tests/core/object/test_object.h

```diff
@@ -298,7 +298,7 @@ TEST_CASE("[Object] Signals") {
 	List<MethodInfo> signals_before;
 	object.get_signal_list(&signals_before);
 
-	object.add_user_signal(MethodInfo("my_custom_signal"));
+	object.add_user_signal(MethodInfo("my_custom_signal")); // This one prevents running remove_user_signal
```

Comment:
This shouldn't be the case. Do you have some time to investigate?

Reply:
Yeah, i know this happens since when i merged it i had added another variable to check if its being done from C++ or from GDScript #90674

Reply:
This would make it only allow to remove signals added with `Object::_add_user_signal`, commented as:
`this version of add_user_signal is meant to be used from scripts or external apis`